examples/theming.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { styled, createThemeProvider } from '@compiled/css-in-js';
+
+export default {
+  title: 'theming',
+};
+
+const ThemeProvider = createThemeProvider();
+
+const Thing = styled.div({
+  fontSize: '20px',
+  color: 'theme(primary)',
```

Comment:
@gwyneplaine one thing im on the fence about is the global theming api isn't typed right now.

any thoughts how we could make it? since the tokens are sourced from a json file we don't have... an easy way to do it i think..